apps/notifications/src/app/note-shortcuts/index.tsx

```diff
@@ -22,15 +21,8 @@ function Shortcut( { letter, title }: { letter: string; title: string } ) {
 }
 
 export default function NoteShortcuts() {
-	// Ensure the component has a focused wrapper container
-	// so that event handlers work when it is rendered inside a <Popover>.
-	const focusRef = useRef< HTMLDivElement >( null );
```

Comment:
I guess we still need this. Otherwise, the panel cannot be closed when the shortcuts popover opens 🤔

Reply:
It's not needed because we have passed `focusOnMount: true` on the parent dropdown. We don't need this because there is no Navigator on this dropdown, so the focus will never be lost.

Could you verify? I tested and it still works.

Reply:
It didn't work on my side 😭

https://github.com/user-attachments/assets/e16c20e3-2fb8-4d13-84e9-180d1baa85c2

Reply:
Sorry you're right!!! I restored it, could you check again 🙈